### ISSUE TITLE
FD can't save a file if it has the "hidden" attribute set

### ISSUE THREAD

**Issue by gene-pavlovsky** [fdorg/flashdevelop#1166]
FlashDevelop can't save an edited file if it has the "hidden" attribute set. (Notepad can save the same file). Not a permission problem: `attrib -h file` and FD can save it again.

```
Access to the path 'C:\project\apps\FXCalc\fonts\.FontLibrary-config.xml' is denied.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   ...
```

**Comment by wise0704** [comment]
I know it's easy to reproduce this, but providing the full stack trace would be better :)
```
Access to the path '{0}' is denied.

   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize)
   at System.IO.StreamWriter..ctor(String path, Boolean append)
   at PluginCore.Helpers.FileHelper.WriteFile(String file, String text, Encoding encoding, Boolean saveBOM)
   at FlashDevelop.Docking.TabbedDocument.Save(String file)
   at FlashDevelop.Docking.TabbedDocument.Save()
   at FlashDevelop.MainForm.Save(Object sender, EventArgs e)
```

**Comment by gene-pavlovsky** [comment]
Sorry I didn't provide the stack trace

**Comment by gene-pavlovsky** [comment]
Verified, it works now. Thanks!

**Comment by gene-pavlovsky** [comment]
I'm suspecting that this fix causes #1270 !
To reproduce, open any file, delete some of it's contents (reduce it's size), and save. The new contents are saved on top of the old contents, but the file size is not reduced. Whatever was at the end of the file, remains there. It doesn't show up in FD until file is closed&reopened.
